databaseconnect.py

```diff
@@ -150,3 +150,54 @@ def learn_question_response(H):
     db.commit()
     B = "Thank you! I have learnt this."
     return B,0
+
+def clear_table(table_name):
+    db = connection_to_database()
+    cur = db.cursor()
+
+    if table_name in ("question_table","statement_table"):
+        tables_to_be_deleted = ["question_table","statement_table"]
+        print("The following tables will be deleted:\n")
+        for table in tables_to_be_deleted:
+            cur.execute("DESC %s" % table)
+            res = cur.fetchall()
+            column_names = [col[0] for col in res]
+
+            cur.execute("SELECT COUNT(*) FROM %s" % table)
+            res = cur.fetchall()
+            records_no = res[0][0]
+
+            print("Table Name:", table)
+            print("Columns:", column_names)
+            print("Number of existing records:", records_no)
+            print()
+
+        if input("Enter 'Y' to confirm deletion of BOTH tables: ") in ("Y","y"):
+            for table in tables_to_be_deleted:
+                cur.execute("DELETE FROM %s" % table)
```

Comment:
Couple of things here -
1. We are not deleting the tables, merely the data, please change the display text. 
2. Please use f-strings to format this string like this `f'DELETE FROM {table}'`

Reply:
Sure. Does *cleaning tables* sound better? Or does *clearing tables*?

Reply:
Cleaning sound better to me